src/store/modules/Overview/OverviewStore.js

```diff
@@ -0,0 +1,33 @@
+import api from '../../api';
+
+const OverviewStore = {
+  namespaced: true,
+  state: {
+    serverInfo: '--'
+  },
+  getters: {
+    serverInfo(state) {
+      return state.serverInfo;
+    }
+  },
+  mutations: {
+    setServerInfo(state, serverInfo) {
+      state.serverInfo = serverInfo;
+    }
+  },
+  actions: {
+    getServerInfo({ commit }) {
+      api
+        .get('/xyz/openbmc_project/inventory/system')
+        .then(({ data }) => {
+          const serverInfo = data.data;
```

Comment:
Are we going to use object destructuring or maintain consistency in showing the response's data structure? I feel like its more readable when not using object destructuring.